docs/modules/ROOT/pages/development.adoc

```diff
@@ -347,6 +347,34 @@ The corrector allows you to `insert_after`, `insert_before`, `wrap` or
 Range can be determined on `node.location` where it brings specific
 ranges for expression or other internal information that the node holds.
 
+==== Preventing Clobbering
```

Comment:
This seemed like a reasonable place to put this, although maybe there's somewhere better.